packages/evm/contracts/budgets/SimpleBudget.sol

```diff
@@ -17,7 +17,7 @@ import {ASimpleBudget} from "contracts/budgets/ASimpleBudget.sol";
 /// @title Simple ABudget
 /// @notice A minimal budget implementation that simply holds and distributes tokens (ERC20-like and native)
 /// @dev This type of budget supports ETH, ERC20, and ERC1155 assets only
-contract SimpleBudget is ASimpleBudget, Ownable, ReentrancyGuard {
+contract SimpleBudget is ASimpleBudget, ReentrancyGuard {
```

Comment:
I'm not sure there's actually a functional difference between this budget and the managed budget now lol

Reply:
lool